pyomo/contrib/pynumero/interfaces/ampl_nlp.py

```diff
@@ -424,6 +424,26 @@ def set_duals_ineq(self, duals_ineq):
     def get_duals_ineq(self):
         return self._duals_ineq.copy()
 
+    # overloaded from NLP
+    def get_obj_scaling(self):
+        return None
+
+    # overloaded from NLP
+    def get_primals_scaling(self):
+        return None
+
+    # overloaded from NLP
+    def get_constraints_scaling(self):
+        return None
+
+    # overloaded from ExtendedNLP
+    def get_eq_constraints_scaling(self):
+        return None
+
+    # overloaded from ExtendedNLP
+    def get_ineq_constraints_scaling(self):
+        None
```

Comment:
I don't see these methods being used.  Is this put in for future-proofing?  If that is the case, does it make sense to omit these methods and only add them when we need to support them?

Reply:
I implemented them. I would rather the API be complete.